op-node/rollup/driver/sequencer.go

```diff
@@ -29,6 +29,10 @@ type SequencerMetrics interface {
 	RecordSequencerReset()
 }
 
+// Sequencing produces unsafe blocks only, and should not interrupt the L2 block building of safe blocks,
+// e.g. when catching up with an L1 chain with existing batch data.
+const safeBuildInterruptBackoff = 5 * time.Second
```

Comment:
I'm a little worried that this backoff is greater than the L2 block time.

Reply:
should the backoff be the blocktime?

Reply:
If there is a safe block being built, it means it's temporarily busy with constructing from L1 data, and we shouldn't try to sequence at the same time. This is more like an error backoff than a request to sync soon, and should mostly just be higher than the expected block building time to not stumble over the same block building job repeatedly. Setting it to the blocktime works too. Will change it, but I think at best it starts sequencing a tiny bit faster after it completes syncing a block from L1.